x-pack/filebeat/module/okta/system/ingest/pipeline.yml

```diff
@@ -4,6 +4,503 @@ processors:
   - set:
       field: event.ingested
       value: "{{_ingest.timestamp}}"
+  - script:
+      # drops null/empty values recursively
+      lang: painless
+      source: |
+        boolean drop(Object o) {
+          if (o == null || o == "") {
+            return true;
+          } else if (o instanceof Map) {
+            ((Map) o).values().removeIf(v -> drop(v));
+            return (((Map) o).size() == 0);
+          } else if (o instanceof List) {
+            ((List) o).removeIf(v -> drop(v));
+            return (((List) o).length == 0);
+          }
+          return false;
+        }
+        drop(ctx);
+  - remove:
+      field: message
+      ignore_missing: true
+  - convert:
+      field: json.uuid
+      target_field: "@metadata._id"
```

Comment:
This is having a different effect in Ingest Node. The purpose of this on the Beats was to set the document `_id`. In Ingest Node this is done by directly setting `_id`. See https://www.elastic.co/guide/en/elasticsearch/reference/master/accessing-data-in-pipelines.html#accessing-metadata-fields.